packages/webapp/src/components/Finances/Transaction/Mobile/List.jsx

```diff
@@ -0,0 +1,94 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import clsx from 'clsx';
+import ExpandableItem from '../../../Expandable/ExpandableItem';
+import ExpandedContent from '../ExpandedContent';
+import Button from '../../../Form/Button';
+import TransactionItem from './Item';
+import useExpandable from '../../../Expandable/useExpandableItem';
+import { useCurrencySymbol } from '../../../../containers/hooks/useCurrencySymbol';
+import styles from './styles.module.scss';
+
+export const MainContent = ({ t, note, typeLabel, amount, icon, currencySymbol }) => {
+  return (
+    <TransactionItem
+      transaction={note || t('FINANCES.TRANSACTION.LABOUR_EXPENSE')}
+      type={typeLabel || t('SALE.FINANCES.LABOUR_LABEL')}
+      amount={amount}
+      iconKey={icon || 'LABOUR'}
+      currencySymbol={currencySymbol}
+    />
+  );
+};
+
+const generateRows = (t, data, currencySymbol) => {
```

Comment:
This is behaving as a React component, so we could make it so more explicitly and render it instead of call it in the TransactionsList component.